cmd/application.go

```diff
@@ -139,6 +144,50 @@ var applicationSetCmd = &cobra.Command{
 	},
 }
 
+var applicationDescribeCmd = &cobra.Command{
+	Use:   "describe",
+	Short: "describe the given application",
+	Run: func(cmd *cobra.Command, args []string) {
+		client := getOcClient()
+		fmt.Println("Application", args[0], " has:")
+
+		componentList, err := component.List(client)
+		if err != nil {
+			fmt.Println(err)
+			os.Exit(1)
+		}
+		currentProject := project.GetCurrent(client)
```

Comment:
Would it make sense to move this to  `application.Describe`. so we use that function in other places like displaying info before the application gets deleted?

The downside might be that `application.Describe` will have to print to stdout.
WDYT @containscafeine?